src/Glpi/Form/AnswersHandler/AnswersHandler.php

```diff
@@ -118,31 +119,6 @@ public function saveAnswers(
         }
     }
 
-    /**
-     * Insert additionnal data into the given raw answers array to be used when rendering
-     *
-     * @param array $answers The raw answers array
-     *
-     * @return array The formatted answers array
-     */
-    public function prepareAnswersForDisplay(array $answers): array
```

Comment:
This method was only needed to instantiate each questions type before using them in a twig template.

The new Answer object has a `getType()` method that can be called directly from the delete, thus this layer is no longer needed.